scout/adapter/mongo/panel.py

```diff
@@ -303,6 +304,27 @@ def gene_to_panels(self, case_obj):
 
         return gene_dict
 
+    def panel_to_genes(self, panel_id):
+        """Return all hgnc_ids for a given gene panel
+
+            Args:
+                panel_id(ObjectId): _id of a gene panel
+
+            Returns:
+                gene_list(list): a list of hgnc terms
+
+        """
+        gene_list = []
+        panel_obj = None
+        try:
+            panel_obj = self.panel(panel_id)
+        except InvalidId:  # Occurrs when clinical filter panel is used
```

Comment:
Because clinical filter is passing gene panels by name, not id. And the latest panel with that name is collected.